Add Mithril end to end instructions for `--skip-cardano-bin-download` command (#1331)
input-output-hk/mithril: mithril-test-lab/mithril-end-to-end/README.md

./mithril-end-to-end --db-directory db/ --bin-directory ../../target/release
```

> > [!WARNING]

Reviewer: Maybe just a **### Note for MacOS users** instead of a **[!WARNING]**.

Can you can also remove all the `>`.

> 

> > For MacOS users (using M1 or newer), the option `--skip-cardano-bin-download` must be added to the previous command.

Reviewer: - What about older than M1 MacOS users?
- IMH this sentence should be removed

> 

> > The cardano binaries downloaded to run the test suite are only compatible with Linux OS.

Reviewer: ```suggestion
> The cardano binaries downloaded to run the test suite are only built for Linux.
```

> 

> > To run the Mithril End To End Tests Suite, you will need to build and use your own compiled cardano binaries:

Reviewer: ```suggestion
> To run the end to end test, you will need to build your own cardano binaries:
```

> 
> - Build the **cardano-node** and **cardano-cli** binaries following the documentation on the [Cardano Developer Portal](https://developers.cardano.org/docs/get-started/installing-cardano-node#macos).
> 

> > - From the root of your Mithril repository, copy the **cardano-node** and **cardano-cli** binaries in the devnet directory:

Reviewer: ```suggestion
> - From the root of the repository, copy the `cardano-node` and `cardano-cli` binaries in the `devnet` directory:
```

> ```bash
> cp $HOME/.local/bin/cardano-cli mithril-test-lab/mithril-devnet/cardano-node
> cp $HOME/.local/bin/cardano-cli mithril-test-lab/mithril-devnet/cardano-cli
> ```
> 

> > - Use the `--skip-cardano-bin-download` option to run the Mithril End To End Tests Suite:

Reviewer: ```suggestion
> - Use the `--skip-cardano-bin-download` option to run the end to end test
```
